index.html

```diff
@@ -217,5 +217,19 @@ <h2>
             </div>
         </section>
         </main>
+
+        <footer> 
+            <section class="contact">
```

Comment:
:red_circle:  Kindly make sure the shapes in the  background appear to match the figma design

Your Design
![yours](https://user-images.githubusercontent.com/26736582/199837717-83e0052c-333f-410c-bc4a-771d907b8786.png)

Figma Design
![shaops](https://user-images.githubusercontent.com/26736582/199838097-923401ee-dcf6-46fd-bf6a-ea016eee13e1.png)


- [ ]  You can watch this :arrow_down:  to download the shapes set
![showstudent](https://user-images.githubusercontent.com/26736582/199838524-c3a36512-e267-439a-b61d-a94c25ee9f2a.gif)